environments/prod/prod.tfvars

```diff
@@ -81,9 +81,6 @@ sds_non_production_subscriptions = {
   DTS-SHAREDSERVICES-TEST = {
     deploy_acme = true
   }
-  DTS-SHAREDSERVICES-SBOX = {
-    deploy_acme = true
-  }
```

Comment:
should probably remove this in a separate PR, it's not related to a diagram

Reply:
Can be done but that will temporarily muck up the diagram. Will be fixed in the subsequent PR if that's ok.

Reply:
why not just fix it before this is merged, when you update this PR there should be no change then?

Reply:
The issue is that for some reason, this sub is in two management groups in the code when it should only be in one. When there's two in the code it makes the diagram look like this:
<img width="265" alt="image" src="https://user-images.githubusercontent.com/80516065/201685238-1703460a-7320-4193-9a78-e013f585658b.png">